imodqgis/cross_section/cross_section_data.py

```diff
@@ -179,32 +215,36 @@ def __init__(self, layer, variables_indexes, variable, layer_numbers):
         self.color_widget = self.unique_color_widget
         self.legend_items = []
         self.styling_data = np.array(self.variables)
+        self.cache = {}
```

Comment:
Maybe write a comment what is being cached here, for what purpose, and when the cache is cleared.